content/en/blog/2022/instrument-kafka-clients/index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Instrumenting Apache Kafka clients with OpenTelemetry
+linkTitle: Instrumenting Apache Kafka clients with OpenTelemetry
+date: 2022-09-05
+author: "[Paolo Patierno](https://github.com/ppatierno)"
+---
+
+Nowadays, [Apache Kafka](https://kafka.apache.org/) is chosen as the central piece in a distributed system.
```

Comment:
> the central piece in a distributed system.

the central piece for what? I as a reader have the feeling there is something missing or should it be "a central piece"?

Reply:
The idea behind this was to say that it's used a the "nervous" system for communication between applications in a distributed system.

Reply:
I would be ok with using that term "nervous system", I guess most people understand that picture.